pyOCD/flash/__init__.py

```diff
@@ -32,10 +33,12 @@
 
 FLASH = {
          'cortex_m' : Flash_cortex_m,
+         'kinetis' : Flash_cortex_m,
```

Comment:
To have a generic Kinetis device, similar to the generic Cortex-M. Since Kinetis devices have different and incompatible flash configurations (and therefore drivers), we have to disable flash support for it.